solidity/contracts/deposit/DepositRedemption.sol

```diff
@@ -108,7 +108,12 @@ library DepositRedemption {
         // Convert the 8-byte LE ints to uint256
         uint256 _outputValue = abi.encodePacked(_outputValueBytes).reverseEndianness().bytesToUint();
         uint256 _requestedFee = _d.utxoValue().sub(_outputValue);
+
         require(_requestedFee >= TBTCConstants.getMinimumRedemptionFee(), "Fee is too low");
+        require(
+            _requestedFee < _d.utxoValue() / 2,
```

Comment:
we could use SafeMath here, but its not necessary

Reply:
Talked about this a bit further. Strictly speaking SafeMath can only give us a clearer error message when dividing by 0; otherwise it's identical to a regular division. Given that we have the divisor set to 2, and it would need to become dynamic and be subject to a 0 value for divide-by-0 to be an issue, we went back and forth on it.

I pondered this a bit long and I think in the interest of time here, I'm going to suggest we punt and merge this as-is @NicholasDotSol . If you're comfortable with that, merging is in your court :)